mainframer.sh

```diff
@@ -31,6 +31,13 @@ COMMON_IGNORE_FILE="$CONFIG_DIR/ignore"
 LOCAL_IGNORE_FILE="$CONFIG_DIR/localignore"
 REMOTE_IGNORE_FILE="$CONFIG_DIR/remoteignore"
 
+ HOURS_TAG="hours"
+ MINUTES_TAG="minutes"
+ SECONDS_TAG="seconds"
+ HOURS_FORMAT="%02d"
+ MINUTES_FORMAT="%02d"
+ SECONDS_FORMAT="%02d"
```

Comment:
Looks like it's not the best format, you can check logs on CI:

```
took 00 minutes 07 seconds
```

https://travis-ci.org/gojuno/mainframer/builds/201833897#L2406